packages/block-library/src/query/edit/enhanced-pagination-modal.js

```diff
@@ -15,7 +15,7 @@ import { useState, useEffect } from '@wordpress/element';
 import { useContainsThirdPartyBlocks } from '../utils';
 
 const disableEnhancedPaginationDescription = __(
-	'Third-party blocks are not supported inside a Query Loop block with enhanced pagination enabled. To re-enable it, remove any third-party block and then update it in the Query Loop settings.'
+	'Plugin blocks are not supported. For enhanced pagination to work, remove the plugin block, then re-enable it in settings.'
```

Comment:
Maybe we could add _yet_ to indicate that they will be supported at some point?

```suggestion
	'Plugin blocks are not supported yet. For enhanced pagination to work, remove the plugin block, then re-enable it in settings.'
```

cc: @jasmussen

Reply:
Seems good. I prefer not to repeat "enhanced pagination" too many times, but "re-enable it" may be too vague without it 🤔

Reply:
I've replaced "it" with "enhanced pagination" in https://github.com/WordPress/gutenberg/pull/54455/commits/e66a31b274c7e506748acbab0b902e15a22f049b.

However, I've realized we're not mentioning the "enhanced pagination" being a setting of the Query Loop block. That could be confusing as the modal is shown when you interact with different blocks (i.e., those being inserted in the Query Loop), and the user could not know where that setting is.

@jasmussen, what do you think? Should we mention the Query Loop somewhere in the modal?

Reply:
I don't think the intricate semantics should block the progress here, so yes, we can try and include it. Though I really do hope in the future we can have a more central location for enabling this good stuff.

Perhaps: 

> Plugin blocks are not supported yet. For the enhanced pagination to work, remove the plugin block, then re-enable "Enhanced pagination" in the Query Block settings.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/54455/commits/4e5760dc47d83a32a3f272fba9c04052dcc1f158.

<img width="520" alt="Screenshot 2023-09-14 at 19 34 06" src="https://github.com/WordPress/gutenberg/assets/6917969/81701857-fb91-40da-be68-0ab083b110ba">